### DISCUSSION TITLE
Dont's

### DISCUSSION THREAD

**Discussion by leminsc8er** [iSoron/uhabits#1065] [Feature Requests]
I wish you will add "dont's". I dont Need a nother App iff you will add this. 

For consuming drugs, gaming, porn and other bad thinks.it will be helpfull  of i see this.sorry for my bad english.

**Comment by ganesshkumar** [comment]
Similar to this feature request? https://github.com/iSoron/uhabits/discussions/1062

**Reply by hiqua** [reply]
Duplicate: https://github.com/iSoron/uhabits/discussions/696
